cron.Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.6-alpine
+
+# Create user with home directory and no password and change workdir
+WORKDIR /api
+# Environment variable for config, use path for docker secrets as default
+ENV AMIVAPI_CONFIG=/run/secrets/amivapi_config
+
+# Install bjoern and dependencies for install (we need to keep libev)
+RUN apk add --no-cache --virtual .deps \
+        musl-dev python-dev gcc git
+
+# Copy files to /api directory, install requirements
+COPY ./ /api
+RUN pip install -r /api/requirements.txt
+
+# Cleanup dependencies
+RUN apk del .deps
```

Comment:
Everything above seems exactly like the normal dockerfile.

Could you not simply use FROM amiveth/amivapi instead to use the normal Dockerfile as starting point to reduce maintenance, if we even need to change something?